server/grpc_service.go

```diff
@@ -1451,7 +1453,7 @@ var mockLocalAllocatorLeaderChangeFlag = false
 
 // SyncMaxTS will check whether MaxTS is the biggest one among all Local TSOs this PD is holding when skipCheck is set,
 // and write it into all Local TSO Allocators then if it's indeed the biggest one.
-func (s *Server) SyncMaxTS(ctx context.Context, request *pdpb.SyncMaxTSRequest) (*pdpb.SyncMaxTSResponse, error) {
+func (s *GrpcServer) SyncMaxTS(ctx context.Context, request *pdpb.SyncMaxTSRequest) (*pdpb.SyncMaxTSResponse, error) {
```

Comment:
This method is used internally also, could you please help move this into `Server`?

Reply:
Sure. I noticed it after pushing the PR.

Reply:
@JmPotato Found that they are not used internally. The `allocator manager` does have a method with the same name.  Maybe you're misremembering.